src/Host/Client/Impl/BrokerServices/WebService.cs

```diff
@@ -32,7 +32,8 @@ public class WebService {
                 if (s != null) {
                     BrokerApiError apiError;
                     if (Enum.TryParse(s, out apiError)) {
-                        throw new BrokerApiErrorException(apiError);
+                        response.Headers.TryGetValues(CustomHttpHeaders.RTVSBrokerException, out values);
```

Comment:
Ok

---
In reply to: [80277735](https://github.com/Microsoft/RTVS/pull/2393#discussion_r80277735) [](ancestors = 80277735)